Attendance_app.py

```diff
@@ -185,9 +191,22 @@ def __init__(self, parent, controller):
                                     fg=self.text_color,
                                     font=("Courier", text_fontsize))
         self.lb_username.place(x=183 * scale_factor, y=210 * scale_factor)
-        self.tv_username = tk.Entry(self, width=entry_width)
-        self.tv_username.place(x=340 * scale_factor, y=215 * scale_factor)
-
+        
+        
+    
+        # Class Codes Autocompletion Stuff 
+        self.filled = False
+
+        self.classCodes=list(class_codes)
+        self.class_variable = StringVar(self)
+        self.class_variable.set('') # default value
+        self.tv_class=tk.Entry(self,textvariable=self.class_variable)
+        self.tv_class.focus_set()
+        self.tv_class.bind('<KeyRelease>', self.get_typed)
+        self.tv_class.bind('<Key>', self.detect_pressed)
+        self.tv_class.place(x=400* scale_factor, y=160* scale_factor)

```

Comment:
Initialization, focussing and placement have been done twice.
Also done on these lines:
https://github.com/Marauders-9998/Attendance-Management-using-Face-Recognition/blob/866f0cf26a3b5b11f0c13d4c08bc211f2e8feef5/Attendance_app.py#L186-L188